bundle/edu.gemini.p1backend/TODO.md

```diff
@@ -0,0 +1,50 @@
+# Short term
+
+Something that improves the situation now. Nothing for the future. A dead-end by
+design. Something to keep chugging away until we do it right. It shouldn't
+take more work than 2 weeks.
+
+1. Code lives in ocs.
+1. URLs:
+   - Receiver of P1 submissions: phase1.gemini.edu/receive/.../...
+   - Viewer of P1 submissions: phase1.gemini.edu/fetch/.../...
+1. Each backend a different path like now.
+1. Configuration file for each backend (that file is read with every new
+   submission). Used for prepending semester to Ids, close deadline, etc.
+1. Id counter from previous file names (hashes discussed but discarded).
+1. Different backend for joints? If not possible use hard links.
+1. Generate PDFs.
+1. Stick together the XML and PDFs in tarballs.
+1. Send notifications *after* everything is successful. Consult configuration
+   file for emails.
+1. Multiple p1 versions:
```

Comment:
It was my understanding that the PIT is backwards compatible with previous proposals created with older PIT versions. That is, it could open proposal from older p1 versions and automatically upgrades it to the newest one. The plan in the backend was to use the same serializer/deserializer that the PIT uses.

Of course, I'm guessing it won't work all that smoothly but this is what I meant when I said that I wanted to see first hand what's exactly is the problem when trying it.